### PR TITLE
fix: missing type exports for `hasInjectionContext`

### PR DESCRIPTION
**Pull request by DamianOsipiuk**
Closes #234

### CODE REVIEW

File: lib/v2.7/index.d.ts [hunk 1]
```diff
@@ -34,3 +34,5 @@ export interface App<T = any> {
 }
 export declare function createApp(rootComponent: any, rootProps?: any): App
 // #endregion
+
+export function hasInjectionContext(): boolean
```
Comment (posva) [inline lib/v2.7/index.d.ts:38]:
```suggestion
export declare function hasInjectionContext(): boolean
```

File: lib/v2/index.d.ts [hunk 2]
```diff
@@ -29,3 +29,5 @@ export {
   version,
   install,
 }
+
+export function hasInjectionContext(): boolean
```
Comment (posva) [inline lib/v2/index.d.ts:33]:
```suggestion
export declare function hasInjectionContext(): boolean
```

**Review by posva — APPROVED** [review]
Thanks! I'm adding the `declare` keyword like others

File: lib/v2/index.d.ts [hunk 3]
```diff
@@ -29,3 +29,5 @@ export {
   version,
```
Comment (aptx4869211) [inline lib/v2/index.d.ts:29]: fff

**Pull request merged**